pkg/identity/utils.go

```diff
@@ -12,8 +12,10 @@ import (
 	"encoding/asn1"
 	"encoding/pem"
 	"io/ioutil"
+	"log"
```

Comment:
Pass in `zap.Logger` rather than relying on the global logger.

Reply:
How? Logging is super weird in this project and I haven't figured it out yet. Is there a good example somewhere?